syn/python/get_kge.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Parse a yosys area report and give a kGE equivalient
+
+import re
+import argparse
+
+weighted_dict = {}
```

Comment:
Eugh, globals! Probably cleaner to return `weighted_dict` from `read_lib` and then pass it as an argument to `get_kge`.

Reply:
Will do